libmachine/mcnutils/crc.go

```diff
@@ -229,8 +231,21 @@ func (b *B2dUtils) CopyDiskToMachineDir(diskURL, machineName string) error {
 
 	// By default just copy the existing "cached" iso to the machine's directory...
 	if diskURL == "" {
-		log.Infof("Copying %s to %s...", b.path(), machineIsoPath)
+		log.Infof("\n\nCopying %s to %s...\n\n\n", b.path(), machineIsoPath)
```

Comment:
why the extra \n ?

Reply:
Guess! :) I was debugging and forgot to remove this change.